templates/base.html

```diff
@@ -1,4 +1,4 @@
-{% load static pages_tags mezzanine_tags fullurl wagtailuserbar %}
+{% load static pages_tags compress wagtailuserbar wagtailcore_tags %}
```

Comment:
i tested locally and iirc this worked without `pages_tags` which is mezzanine, fyi